src/braket/circuits/composite_operator.py

```diff
@@ -11,6 +11,19 @@ class CompositeOperator(QuantumOperator):
     """
 
     def __init__(self, qubit_count: int, ascii_symbols: Sequence[str]):
+        """
+        Args:
+            qubit_count (int): Number of qubits this composite operator interacts with.
+            ascii_symbols (str): ASCII string symbols for the composite operator. These are used
+                when printing a diagram of circuits.
+
+        Raises:
+            ValueError: `qubit_count` is less than 1, or `ascii_symbols` are `None`
+        """
+        if len(ascii_symbols) != 1:
```

Comment:
This is fine for now, but when we merge into the main repository it'll be worth thinking about how this can be changed to allow more ascii symbols.